PubSub/src/Topic.php

```diff
@@ -182,6 +182,95 @@ public function create(array $options = [])
         return $this->info;
     }
 
+    /**
+     * Update the topic.
+     *
+     * Note that the topic's name and kms key name are immutable properties and may not be modified.
+     *
+     * Example:
+     * ```
+     * $topic->update([
+     *     'messageStoragePolicy' => [
+     *         'allowedPersistenceRegions' => ['us-central1']
+     *     ]
+     * ]);
+     * ```
+     *
+     * ```
+     * // Updating labels with an explicit update mask
+     * $topic->update([
+     *     'labels' => [
+     *         'foo' => 'bar'
+     *     ]
+     * ], [
+     *     'updateMask' => [
+     *         'labels'
+     *     ]
+     * ]);
+     * ```
+     *
```

Comment:
Could we please add a link to the upstream documentation for the call?

```
@see https://cloud.google.com/pubsub/docs/reference/rest/v1/UpdateTopicRequest Update Topic
```

Reply:
Added.